api/views/teacher.py

```diff
@@ -37,5 +38,7 @@ class PublicTeacherWithPersonalInfoViewSet(viewsets.ReadOnlyModelViewSet[Teacher
     # TODO permissions?
     # TODO filter by coordinator
```

Comment:
remove this TODO?

Reply:
I'm not sure why I wrote it. Do we need an option to filter teachers by their coordinator?

Reply:
No, not really. Just show the teachers for the coordinator that is logged in.